arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1068,17 +1084,9 @@
           "readOnly": true
         },
         "provisioningState": {
```

Comment:
Please add values you currently support. Include both the terminal and non terminal states. You can add more values later when your service supports. Either way, it will be a breaking change later.